Formula/c/cloudsplaining.rb

```diff
@@ -0,0 +1,114 @@
+class Cloudsplaining < Formula
+  include Language::Python::Virtualenv
+
+  desc "AWS IAM Security Assessment tool"
+  homepage "https://cloudsplaining.readthedocs.io/en/latest/"
+  url "https://files.pythonhosted.org/packages/36/41/10330ab113c1d40a48f32f536579aff5025ea975098f37def13fd7fdaef6/cloudsplaining-0.6.2.tar.gz"
+  sha256 "02fc4f8482b701b0b19b7ad543b579ba50f31b3e0389bb9f6973391891d6ecd1"
+  license "BSD-3-Clause"
+  head "https://github.com/salesforce/cloudsplaining.git", branch: "master"
+
+  depends_on "python-certifi"
+  depends_on "python-click"
+  depends_on "python-markdown"
+  depends_on "python-markupsafe"
+  depends_on "python@3.11"
```

Comment:
maybe build with python@3.12?

Reply:
Good call. I used `python@3.11` as force of habit 😅